plugins/browser-plugin-youtube-tracking/src/index.ts

```diff
@@ -126,10 +122,14 @@ export function trackingOptionsParser(mediaId: string, conf?: MediaTrackingOptio
 }
 
 export function enableYouTubeTracking(args: { id: string; options?: MediaTrackingOptions }) {
+  if (!Object.keys(_trackers).length) {
+    LOG.error('You might have forgotten to initialize the YoutubeTrackingPlugin on your tracker configuration.');
```

Comment:
nit: I try and keep strings as short as possible (as they don't minify). This is a little shorter:

```suggestion
    LOG.error('Check YoutubeTrackingPlugin is initialized in tracker config');
```